static/js/search.js

```diff
@@ -33,30 +45,36 @@
     }
 
     window.renderBingSearchResults = () => {
-        var searchTerm  = window.location.search.split("=")[1].split("&")[0].replace(/%20/g,' '),
-            page        = window.location.search.split("=")[2],
-            q           = "site:kubernetes.io " + searchTerm;
+      let urlParams = new URLSearchParams(window.location.search);
+      let searchTerm = urlParams.get("q") || "";
+      let page = urlParams.get("page") || 1;
+      let q = searchTerm;
+      let results = '';
+      let offset = (page - 1) * 10;
+      let ajaxConf = {};
 
-        page = (!page) ?  1 : page.split("&")[0];
+      if (!searchTerm) return;
 
-        var results = '', pagination = '', offset = (page - 1) * 10, ajaxConf = {};
+      ajaxConf.url = 'https://kubernetes-io-search.azurewebsites.net/api/bingsearchproxy';
```

Comment:
(for reference of reviewers)
Here's the api change which LGTM

Reply:
Who runs https://kubernetes-io-search.azurewebsites.net, and does that SIG know about this change?

Reply:
I think that would be [SIG/Network](https://github.com/kubernetes/community/tree/master/sig-network#contact),
Should we ask for a technical review?

Reply:
The CNCF run and pay for https://kubernetes-io-search.azurewebsites.net/ - context: I work for CNCF.